SpaceCadetPinball/SpaceCadetPinball.cpp

```diff
@@ -27,7 +27,7 @@ int main(int argc, char* argv[])
 }
 
 #if _WIN32
-#include <windows.h>
+#include <Windows.h>
```

Comment:
This won’t compile with mingw.
The opposite of what static analyzer is preaching here.
Fool me once... :)